app/models/course/group_user.rb

```diff
@@ -0,0 +1,17 @@
+class Course::GroupUser < ActiveRecord::Base
+  stampable
+  belongs_to :user, inverse_of: :course_group_users
+  belongs_to :group, class_name: Course::Group.name, inverse_of: :group_users,
```

Comment:
As we discussed, both of <code>group_user</code> and <code>group</code> are scoped under course, why still <code>course_group</code> ?